services/models/destination_policy.go

```diff
@@ -4,10 +4,32 @@ import (
 	"github.com/kiali/kiali/kubernetes"
 )
 
+// DestinationPolicies destinationPolicies
+//
+// This is used for returning an array of DestinationPolicies
+//
+// swagger:model destinationPolicies
+// An array of destinationPolicie
+// swagger:allOf
 type DestinationPolicies []DestinationPolicy
+
+// DestinationPolicy destinationPolicy
+//
+// This is used for returning a DestinationPolicy
+//
+// swagger:model destinationPolicy
 type DestinationPolicy struct {
-	Name            string      `json:"name"`
-	CreatedAt       string      `json:"createdAt"`
+	// The name of the destinationPolicy
+	//
+	// required: true
+	Name string `json:"name"`
+	// The creation date of the destinationPolicy
```

Comment:
Good if fields with format expected can define a note about this.
"createdAt should be a string with format XYZ"
Also, when field is a string, but it is expected a known value, potential values should be described.

This comment is not to fill all the fields, but to enable swagger to add these kind of info (we can fix them on future iterations).

For example the Severity field on this one it can accept "LOW, MEDIUM, HIGH, CRITICAL":
http://www.hawkular.org/docs/rest/rest-alerts-v2.html#Trigger

A potential 400 error can be send back if there is some incorrect value.

Basically we need to document these cases as well.

Reply:
Yes we can set // pattern: \w[\w-]+